### PR TITLE
Update script to copy specs from main repo to their appropriate location in specs repo

### PR DESCRIPTION
**Pull request by wmathurin**
E.g. to get the specs from unstable and use them for 4.0.1 do:
update.sh -b unstable -v 4.0.1

### CODE REVIEW

File: update.sh [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+      	  exit 2
+    fi
+
+}
+
+# Helper functions
+get_root_folder ()
+{
+    local current_folder=`dirname "${BASH_SOURCE[0]}"`
+    echo `cd $current_folder && cd .. && pwd`
```
Comment (khawkins) [inline update.sh]: This is not valid for this script, which already lives at the root.

Comment (khawkins) [inline update.sh]: suggest: `echo $(cd $current_folder && pwd)`

**Comment by wmathurin** [comment]
@khawkins done. Thought I had made the change - it worked locally (by leveraging the iOS repo I had checked out one level up).

**Comment by khawkins** [comment]
Looks fine otherwise. :+1:

**Pull request merged**